addons/pos_sale/views/pos_config_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_pos_config_form_inherit_pos_sale" model="ir.ui.view"> 
```

Comment:
yeah but I inherit a badly named xml_id of that name :P 

Do I rename the original one ? If not I believe it's better to keep the inherited name.